venue.md

```diff
@@ -2,26 +2,12 @@
 title: Venue
 ---
 
-LambdAle will be held in the ballroom at [Balham Bowls Club](http://balhambowlsclub.com/).
+LambdAle will be held in the ballroom at [The George IV Pub](https://www.georgeiv.co.uk).
 
-The address is [7-9 Ramsden Rd, London SW12 8QX](https://www.google.com/maps/place/Balham+Bowls+Club/@51.444971,-0.152492,16z/data=!4m5!3m4!1s0x0:0x4928317423696e4d!8m2!3d51.4449078!4d-0.1525026?hl=en-GB).
+The address is [185 High Road, Chiswick, London, W4 2DR](https://goo.gl/maps/2kHAdvHpHqJ2).
 
-## Map
-
-<iframe src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d19893.772141396432!2d-0.152503!3d51.444908!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0x4928317423696e4d!2sBalham+Bowls+Club!5e0!3m2!1sen!2suk!4v1518043181690" width="300" height="300" frameborder="0" style="border:0" allowfullscreen></iframe>
+<iframe src="https://www.google.com/maps/embed?pb=!4v1550427124320!6m8!1m7!1sCAoSLEFGMVFpcE8yekNEajNfWlJDUjM5OXV4OXZrMWpjX2R0dkdUQ00zekd5cXI0!2m2!1d51.49242125220601!2d-0.2570486842103037!3f168.99!4f8.120000000000005!5f0.4000000000000002" width="600" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>
 
 ## Directions
 
-From central London, either take the Northern line southbound or take National Rail from London Victoria towards Sutton (usually platform 1 or 2).
```

Comment:
The nearest tube stop is Turnham Green.

From central London, take the District line westbound towards Ealing Broadway or Richmond.

Turn left out of Turnham Green station and walk to the end of the road. Turn right and walk down Chiswick High Street for about 100 metres. The pub is on the left hand side.

Somebody will be at the door to greet you. The conference is in the Boston room at the back of the pub.